types/testingutils/beacon_node.go

```diff
@@ -515,30 +467,21 @@ func (bn *TestingBeaconNode) SubmitBeaconBlock(block *spec.VersionedBeaconBlock,
 	var r [32]byte
 
 	switch block.Version {
-	case spec.DataVersionPhase0:
-		if block.Phase0 == nil {
-			return errors.Errorf("%s block is nil", block.Version.String())
-		}
-		sb := &phase0.SignedBeaconBlock{
-			Message:   block.Phase0,
-			Signature: sig,
-		}
-		r, _ = sb.HashTreeRoot()
-	case spec.DataVersionAltair:
-		if block.Altair == nil {
+	case spec.DataVersionBellatrix:
+		if block.Bellatrix == nil {
 			return errors.Errorf("%s block is nil", block.Version.String())
 		}
-		sb := &altair.SignedBeaconBlock{
-			Message:   block.Altair,
+		sb := &bellatrix.SignedBeaconBlock{
```

Comment:
if we're deprecating support for `phase0` and `altair` here, why leave bellatrix?

Reply:
I don't think we really have any tests that use Altair.
We do have tests that use Bellatrix though